specification/datadog/resource-manager/Microsoft.Datadog/stable/2023-01-01/datadog.json

```diff
@@ -142,7 +142,7 @@
           "200": {
             "description": "Success",
             "schema": {
-              "$ref": "#/definitions/CreateResourceSupportedResponseList"
+              "$ref": "#/definitions/CreateResourceSupportedResponse"
```

Comment:
>CreateResourceSupportedResponse [](http://example.com/codeflow?start=37&length=31)

why is this not a list anymore?

Reply:
We do not have multiple values here. Same has been reverted in private repo and we want to be in sync with it.
https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/datadog/resource-manager/Microsoft.Datadog/stable/2023-01-01/swagger.json